docs/api.md

```diff
@@ -671,12 +671,68 @@ const result = await page.evaluate(() => {
 console.log(result); // prints "56"
 ```
 
+Passing parameters (serializable object) to pageFunction.
+
+```js
+const name = 'Thomas';
+const age  = 29;
+const location = 'Paris';
+
+const information = await page.evaluate(({name, age, location}) => {
+  return {
+    nameValue: name,
+    ageValue: age,
+    locationValue: location
+  };
+}, {name, age, location});
+
+console.log(information); // prints "{ nameValue: 'Thomas', ageValue: 29, locationValue: 'Paris' }"
+```
+
+Passing parameters (non serializable object) to pageFunction.
+
+```js
+const name = 'Thomas';
+const age  = 29;
+const location = 'Paris';
+
+const information = await page.evaluate((name, age, location) => {
+  return {
+    nameValue: name,
+    ageValue: age,
+    locationValue: location
+  };
+}, {name, age, location});
+
+console.log(information); // prints: "{ nameValue: { name: 'Thomas', age: 29, location: 'Paris' }, ageValue: null, locationValue: null }"
+```
+
+Passing pageFunction to page.evaluate as parameter.
+```js
+function manipulateDOM(){
```

Comment:
you can pass arguments here exactly in the same fashion: 
```js
function manipulateDOM(arg) {
  return 'puppeteer was here and received ' + arg;
}

const information = await page.evaluate(manipulateDOM, 42);
console.log(information);

Reply:
@aslushnikov this is exactly what I was missing. I've amended the commit. Thank you!